pyad/forward_diff.py

```diff
@@ -505,6 +505,26 @@ def exp(tensor):
     """
     return _elementary_op(tensor, np.exp, np.exp)
 
+def sigmoid(tensor):
```

Comment:
In my other PR (https://github.com/fire-breathing-rubber-lemons/cs207-FinalProject/pull/36/files#diff-6597493df0f57f9dce7db1086271c922R739), I actually already added this function, but I called it `logistic`. So you could probably just remove this function from forward_diff.py